tests/functional/services/api/imports/test_import.py

```diff
@@ -0,0 +1,266 @@
+import json
+
+import pytest
+from anchore_engine.subsys import logger
+from tests.functional.services.api import imports
+from tests.functional.services.utils.http_utils import (
+    APIResponse,
+    http_del,
+    http_get,
+    http_post,
+)
+
+logger.enable_test_logging(level="info")
+
+
+@pytest.fixture
+def module_path(request):
+    return request.module.__file__
+
+
+@pytest.fixture()
+def begin_import() -> APIResponse:
+    logger.info("Creating import operation.")
+    import_response = http_post(["imports", "images"], payload={})
+    yield import_response
+    logger.info("Deleting import operation.")
+    delete_response = http_del(["imports", "images", import_response.body["uuid"]])
+    if delete_response.code == 200:
+        logger.info("Deletion successful.")
+    else:
+        logger.info(f"Deletion failed! Response: {delete_response.body}")
+
+
+@pytest.mark.incremental
+@pytest.mark.parametrize(
```

Comment:
this way we can parametrize the entire class with different syft-sbom+dockerfile combinations in the future.